src/main/java/edu/ucla/library/iiif/fester/CsvParser.java

```diff
@@ -41,6 +41,8 @@
 
     private static final Pattern EOL_PATTERN = Pattern.compile(".*\\R");
 
+    private static final String AV_URL_STRING = "pairtree";
```

Comment:
Where does this value come from? https://docs.library.ucla.edu/pages/viewpage.action?pageId=237784030#DLCSA/VAssetDocumentation-StructureoftheStreamingDirectories (`pairtree_root`)?

Reply:
Hard coding this value to something in our AV URL path prevents others from possibly using the application. I'd suggest making the A/V server base URL configurable and comparing that configured value to the found URL.